CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Main (unreleased)
 ### Enhancements
 
 - Add [godeltaprof](https://github.com/grafana/godeltaprof) profiling types (`godeltaprof_memory`, `godeltaprof_mutex`, `godeltaprof_block`) to `pyroscope.scrape` component
-
+- Update jsonnet libraries to use the `v1` CronJob API instead of the `v1beta1` that is removed from Kubernetes 1.25. The `v1` CronJob has been around since Kubernetes 1.21, making it a safe change breakage-wise.
```

Comment:
I don't we need changelog in this case as it not affecting users.

Reply:
Let me revert this change then. This was added as a precaution in case some of the users would run ancient Kubernetes versions.